sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -428,3 +428,34 @@ def test_importance_getter(estimator, importance_getter):
     )
     selector.fit(data, y)
     assert selector.transform(data).shape[1] == 1
+
+
+class RandomForestNoFeatureNames(RandomForestClassifier):
+    def fit(self, X, y):
+        super().fit(X, y)
+        # Remove feature names
+        del self.feature_names_in_
+        return self
+
+
+def test_estimator_does_not_support_feature_names():
```

Comment:
Could we make a more general test where we iterate over all possible estimators that are inheriting from `MetaEstimatorMixin` and create a classifier or a regressor that ensure the behaviour with the minimal classifier or regressor?

Reply:
Updated test in `sklearn/tests/test_common.py` to generate meta-estimators with MinimalEstimators.

I left this one here to test `get_feature_names_out`.